.github/workflows/fetch-configlet.yml

```diff
@@ -0,0 +1,44 @@
+name: Fetch configlet
+
+on:
+  push:
+    paths:
+      - scripts/fetch-configlet*
+  pull_request:
+    paths:
+      - scripts/fetch-configlet*
+  workflow_dispatch:
+
+jobs:
+  fetch_configlet:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [linux, macOS, windows]
+
+    name: fetch-configlet-${{ matrix.os }}
+    runs-on: ${{ matrix.os == 'linux' && 'ubuntu-22.04' ||
+                (matrix.os == 'macOS' && 'macos-12' || 'windows-2022') }}
```

Comment:
What is the reason you do it like this an not have the matrix specifc the runners? That you can use `matrix.os == 'windows'` later on?

Reply:
See https://github.com/exercism/configlet/pull/692/files/5e7dffebd8f7490df372d3fd4cbd05e03416a2b9#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957f for my thoughts